### DISCUSSION TITLE
Compatibility with Matrix 1.5

### DISCUSSION THREAD

**Discussion by Zepeng-Mu** [GreenleafLab/ArchR#1787] [Feature Requests]
Since this not a bug, I decided to put it in Discussion.
Current version of ArchR is not compitable with the recently released Matrix version 1.5 (such as `addDeviationMatrix()`), whereas many other packages have moved to require Matrix >= 1.5.0.
So while doing analysis, I have to choose not to use certain packages or even switching between various versions by reinstalling.
It would be great if ArchR could support Matrix 1.5 and above.

Thanks!!

**Comment by rcorces** [comment]
Is there something specific that is incompatible that you can point to?

**Reply by Zepeng-Mu** [reply]
One thing I encountered was `Error: as(, "dgCMatrix") is deprecated since Matrix 1.5-0`.

**Comment by rcorces** [comment]
I've transferred this to an Issue as it will require some code changes. Progress can be tracked at #1793

**Reply by rcorces** [reply]
@Zepeng-Mu - fixed on `dev` as detailed in #1793 
Thanks for bringing this to my attention
